File: 완전탐색/2309.js
function getIJ(input){
    ij=[];
    for(i = 0;i<8;i++){
        for(j = i+1; j<9;j++){
            if(total(i,j,input)==100){
                ij.push(i);
                ij.push(j);
                return ij;    
            }
        }
    }
}

function total(a1,a2,arr){
    sum = 0;
    for(x = 0; x<9; x ++){
        if(x != a1 && x != a2){
            sum += arr[x];
        }
    }
    return sum;
}

answer = [];
ij = getIJ(input);
i = ij[0];
j = ij[1];
for(a = 0; a<9; a++){
    if(a != i && a!= j){
        answer.push(input[a]);
    }
}
answer.sort(function(a,b){return a-b;});
console.log(answer);

File: 완전탐색/2309.py
#백준 2309 - 일곱 난쟁이
# https://www.acmicpc.net/problem/2309
import sys
input = sys.stdin.readline

arr = []
def getIJ(arr):
    for i in range(8):
        for j in range(i+1,9):
            if total(i,j,arr) == 100:
                return (i,j)

def total(a1,a2,arr):
    sum = 0
    for i in range(9):
        if(i != a1 and i != a2):
            sum += arr[i]
    return sum

answer = []
ij =getIJ(arr)
i = ij[0]
j = ij[1]
for a in range(9):
    if a != i and a != j:
        answer.append(arr[a])
answer.sort()
#print()
for x in answer:
    print(x)
                


##############################################

# for _ in range(9):
#     arr.append(int(input()))

# total = sum(arr)
# arr.sort()
# for i in range(9):
#     for j in range(i+1,9):
#         if (total - arr[i] - arr[j]) == 100:
#             for k in range(9):
#                 if k !=i and k !=j:
#                     print(arr[k])
#             exit()